toolkit-distribution/src/release/licenses/LICENSE-mit

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c)
```

Comment:
Can you check which project use this license? It doesn't have the Copyright notice.

Reply:
org.checkerframework:checker-compat-qual:2.5.2 use MIT license
The mvn repository: https://mvnrepository.com/artifact/org.checkerframework/checker-compat-qual/2.5.2
And here is the LICENSE file in the code repo: https://github.com/typetools/checker-framework/blob/checker-framework-2.5.2/LICENSE.txt
But the LICENSE is mixed with GPL and MIT, and the MIT part doesn't have copyright information.
I've checked the the checker-compact-qual is under MIT license.

Another dependency under MIT license without copyright information is org.codehaus.mojo:animal-sniffer-annotations:1.17
The mvn repo is here: https://mvnrepository.com/artifact/org.codehaus.mojo/animal-sniffer-annotations/1.17
Here is the project license: http://www.mojohaus.org/animal-sniffer/animal-sniffer-annotations/license.html

Reply:
Thanks for the information. We just keep the License file there.